tests/test_update.py

```diff
@@ -18,6 +18,7 @@
 
 @pytest.mark.parametrize("package_name,branch", TEST_PROJECTS_WITH_BRANCHES)
 def test_update(tmp_path: Path, package_name, branch):
+    """ perform an update from the first dist-git commit to the last one """
```

Comment:
Isn't this `HEAD~1` so last but one?

Reply:
I should definitely go and get my eyes checked @_@ I swear I saw this is as `HEAD-1`

thanks for the pointer